src/components/phylogeny/Phylotree.vue

```diff
@@ -324,169 +359,213 @@ export default {
           const wrappedPhylogeny = new PhylogenyWrapper(this.phylogeny || {});
 
           // Wrap the phyloref is there is one.
-          const wrappedPhyloref = new PhylorefWrapper(this.phyloref || {});
+          this.phylorefs.forEach((phyloref) => {
+            const wrappedPhyloref = new PhylorefWrapper(phyloref || {});
 
-          // Make sure we don't already have an internal label node on this SVG node!
-          let textLabel = element.selectAll("text");
+            // Make sure we don't already have an internal label node on this SVG node!
+            let textLabel = element.selectAll("text");
```

Comment:
Are you sure the indentation in this and the preceding line is correct? It looks like there's extraneous space, but perhaps that's an artifact from tabs being used?

Reply:
That is odd, but yes, it looks like the alignment is correct:

https://github.com/phyloref/klados/blob/cbcf4ea8880ebc3cfb67f53fb82db14bd73db74d/src/components/phylogeny/Phylotree.vue#L363-L368